Week3/homework/index.js

```diff
@@ -12,10 +12,10 @@ const select = document.getElementById('select');
 async function fetchJSON(url) {
   try {
     const res = await axios.get(url);
-    const data = res.data;
-    if (res.status > 299) {
-      throw Error('Failed to fetch!');
+    if (!res.ok || res.status > 299) {
```

Comment:
the ok part was a feature from fetch, two different libraries, two different concepts, that's the lesson of using external library 🙂 
```suggestion
    if (res.status > 299) {
```

Reply:
Well, I did it like you requested but interestingly it did not throw error when the url was like : 

https://api.github**_x_**.com/orgs/HackYourFuture/repos?per_page=100 . So I tried many things then joined Noer's QA session. Then we tried to fix this and ended up to this. Even more strange thing is now it works good with my first version which also your suggestion :D .